app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -18,6 +18,22 @@ def github
     end
   end
 
+  def google
```

Comment:
Can we add unit tests for this method please? they will be pretty much the same as the ones for the `github` method